server/packet_handlers.py

```diff
@@ -349,6 +349,37 @@ async def spectate_frames_handler(session: "Session", packet_data: bytes):
 # CANT_SPECTATE = 21
 
 
+@bancho_handler(packets.ClientPackets.CANT_SPECTATE)
+async def cant_spectate_handler(session: "Session", packet_data: bytes):
+    assert session["presence"] is not None
+    
+    packet_reader = packets.PacketReader(packet_data)
+    host_account_id = packet_reader.read_i32()
```

Comment:
this packet should not accept any parameters - the "cant spectate" action comes from a client who has already started spectating a user, so the server already knows this information - this will raise an error at runtime see reference code below for an example & test the code with the osu client

https://github.com/osuAkatsuki/bancho.py/blob/master/app/api/domains/cho.py#L1092-L1106